go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/cheggaaa/pb/v3 v3.1.4
 	github.com/containerd/containerd v1.7.11
 	github.com/csaf-poc/csaf_distribution/v3 v3.0.0
-	github.com/docker/docker v24.0.7+incompatible
+	github.com/docker/docker v25.0.0+incompatible
```

Comment:
Is there a need to upgrade the docker version? `oras.land/oras-go` is not compatible with this version.